app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC, abstractmethod
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
+    def __init__(self, nickname, favourite_dish):
+        super().__init__(nickname)
+        self._favourite_dish = favourite_dish
+
+    @abstractmethod
```

Comment:
class Dwarf should only have method that prints "{nickname} is eating {favourite_dish}"